### PR TITLE
* FAVICON!

### PR DESCRIPTION
**Pull request by dudemeister**
:)

### CODE REVIEW

**Comment by dudemeister** [comment]
@tiff I MADE THIS!

File: public/index.html [hunk 1]
```diff
@@ -25,8 +25,11 @@
   $.ajax("status", {
     success: function(data) {
       $(document.body).removeClass("occupied free");
+      data.status = "free"
       if (data.status == "free" || data.status == "occupied") {
         $(document.body).addClass(data.status);
+        $("#favicon-shortcut").attr("href","/favicon-" + data.status + ".ico");
+        $("#favicon-icon").attr("href","/favicon-" + data.status + ".ico");
```
Comment (tiff) [inline public/index.html]: One liner:

``` javascript
$("#favicon-icon, #favicon-shortcut").attr("href","/favicon-" + data.status + ".ico");
```

File: public/index.html [hunk 2]
```diff
@@ -25,8 +25,11 @@
   $.ajax("status", {
     success: function(data) {
       $(document.body).removeClass("occupied free");
+      data.status = "free"
```
Comment (tiff) [inline public/index.html]: remove this line

**Comment by dudemeister** [comment]
Good?

**Pull request merged**